### PR TITLE
issue_489 updated udp queries

### PR DESCRIPTION
**Pull request by zqian**
-  retrieve cross listed course enrollments
-  simplified submission query

Test plan:

- setup environment to run `scripts/data_validation/validate_udw_vs_upd.py`, following the instructions inside `docs/testing.md`

### CODE REVIEW

File: config/cron.hjson [hunk 1]
```diff
@@ -165,7 +165,7 @@
             )
             select 
                 f.id, 
-                f.assignment_id, 
+                cast(f.assignment_id as bigint),
```
Comment (pushyamig) [inline config/cron.hjson:168]: Was there any reason this change was made?

**Comment by pushyamig** [comment]
I ran 2 tests for testing this 

1. 2 different courses one with SIS only section and one with NON-SIS section, I see it is pulling the correct roster information.
2. Use the validation script, but I see that for the `comparing query submission`, there was a difference found. this doesn't mean a failure, But I might not know full details of this. 

The courses I was using ["534433", "518589"],

File: config/cron.hjson (hunk 1 above)
Comment (zqian) [reply config/cron.hjson:168]: This is to have the same type as the assignment_id field from the udp query. 

Plus the assignment table inside MyLA database already have id field as bigint(20) type.

**Comment by zqian** [comment]
Thanks for verifying, @pushyamig 

I checked the 534433 course, the submission records are diff for two users, and both got 13.95 in one submission. UDW reported the score as 13.9 vs UDP reported the score as 14. I think this is acceptable.

As for the 518589 course, the validation script did not complain about record difference.

**Review by pushyamig — APPROVED** [review]
I think this change seem to be working.

**Pull request merged**